back-end/src/cron/cron.service.ts

```diff
@@ -0,0 +1,86 @@
+import { Injectable } from '@nestjs/common';
+import { Cron } from '@nestjs/schedule';
+import { CallsRepository } from '@/base/calls/calls.repository';
+import { AbsencesParams } from '@/base/calls/interfaces/calls.interface';
+import { MaijetTemplate } from 'src/common/providers/interfaces/events.interface';
+
+@Injectable()
+export class CronService {
+	constructor(private readonly callsRepository: CallsRepository) {}
+
+	@Cron('0 18 * * *')
+	async handleCron() {
+		if (this.isWeekend()) {
```

Comment:
Pretty sure you can manage that directly on the cron schedule.
@Cron('0 18 * * 1-5')

Reply:
[Cron guru](https://crontab.guru/#0_18_*_*_1-5)

Reply:
Nice catch ! That's my bad didn't think about this